core/admin/mailu/models.py

```diff
@@ -623,11 +624,13 @@ def check_password(self, password):
             self._credential_cache[self.get_id()] = (self.password.split('$')[3], passlib.hash.pbkdf2_sha256.using(rounds=1).hash(password))
         return result
 
-    def set_password(self, password, raw=False):
+    def set_password(self, password, raw=False, session=None):
         """ Set password for user
             @password: plain text password to encrypt (or, if raw is True: the hash itself)
         """
         self.password = password if raw else User.get_password_context().hash(password)
+        if session:
```

Comment:
This is okay, but adding `is not None` would be more precise.

Reply:
done in 786da5e0